modules/mri_violations/test/TestPlan.md

```diff
@@ -22,3 +22,6 @@ MRI Violations Module Test Plan:
     patient if permission 'Across all sites view Dicom Archive modile and pages' is granted
 12. Upload scan that violates protocol and ensure it shows up in the module
 13. Upload scan that violates protocol check and ensure it shows up in the module
+14. Resolve issues with each of the different types of resolutions, and ensure that they show up in the resolved tab, and are no longer in the unresolved tab
```

Comment:
Add checks for permissions constraints: 
* User can only resolve (or edit otherwise) an issue if and only if they have violated_scans_edit permission
* User can resolve issues from their site as well as other sites if and only if they have (superuser?) ...